Sources/Purchasing/Purchases.swift

```diff
@@ -1822,6 +1839,12 @@ extension Purchases: PurchasesOrchestratorDelegate {
 
 public extension Purchases {
 
+    /**
+     * Enable automatic collection of Apple Search Ads attribution. Defaults to `false`.
+     */
+    @available(*, deprecated, message: "use Purchases.automaticAdServicesAttributionTokenCollection for iOS 14.3+ instead")
+    @objc static var automaticAppleSearchAdsAttributionCollection: Bool = false
```

Comment:
Maybe move this to `Deprecations.swift`?

Reply:
is there a difference between methods in Deprecations.swift and those in the // MARK: Deprecated section of Purchases.swift?

Reply:
Oh I didn't realize we still had some there. The idea was to make the main file (which is already huge) smaller by only having the current API.

Reply:
makes sense to me! i can move the rest while i'm at it...